src/api/apiMethods.js

```diff
@@ -153,39 +233,63 @@ const getApiMethodsSelector = () =>
         return columnGroup;
       };
 
-      const getColumn = (
-        columnIndex,
-        cellGroupType = CellGroupType.SCROLLABLE
-      ) => {
-        const column = _getColumn(columnIndex, cellGroupType);
-        const offset = getColumnOffset(columnIndex, cellGroupType);
+      const getColumn = (columnIndex) => {
+        const cellGroupType = _getCellGroupTypeFromColumnIndex(columnIndex);
+        const localColumnIndex = _getColumnLocalIndex(
+          columnIndex,
+          cellGroupType
+        );
+        const column = _getColumn(localColumnIndex, cellGroupType);
+        const offset = _getColumnOffset(localColumnIndex, cellGroupType);
```

Comment:
ditto from https://github.com/schrodinger/fixed-data-table-2/pull/667/files#r1065189535
> Since the client is always gonna call these APIs in terms of the global column index, how about we only convert from global to local indices at the place where it's actually required?
> i.e, _getColumn should be the one who's doing the conversion since it directly uses the internal column lists that require local indices.

Reply:
This _getColumn() is used by multiple APIs, and may require significant refactor now.
Is there  any significant advantage of doing that ?

While refactoring these api, i find it easy to keep all such existing helpers as it is (i.e let them use local index), and just made small change to convert global to local index in apis.

Reply:
The main reason would be abstraction and locality.

The exposed API methods won't have to deal with 2 types of indices.
This can be problematic especially if we have API methods that might rely on each other and then we need to remember what functions require what type of index.

Moreover, the transformation is done at the same place where it's used, and doesn't spill outside, making the code easier to understand.

Reply:
Thanks for explanation,   made the changes so that exposed apis only uses global index